frontend/test/pytest/test_transform.py

```diff
@@ -47,6 +48,50 @@
 from catalyst.utils.exceptions import CompileError
 
 # pylint: disable=unnecessary-lambda-assignment
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+
+
+@pytest.mark.xfail(reason="Noise models not supported on Lightning.")
```

Comment:
How we make sure that these tests are correct? For instance, there is no typo or other types of bugs in them? Have they been run locally with `default.qubit`?

Reply:
They are tested locally with `default.qubit`.